cores/esp8266/esp8266_undocumented.h

```diff
@@ -204,8 +218,22 @@ extern fn_c_exception_handler_t _xtos_c_handler_table[XCHAL_EXCCAUSE_NUM];
   `#ifdef __cplusplus`.
 */
 extern fn_c_exception_handler_t _xtos_set_exception_handler(int cause, fn_c_exception_handler_t fn);
+
+extern void window_spill_exc_handler(struct __exception_frame* ef);
```

Comment:
The core on the 8266 doesn't have register windows implemented, why is this added?

Reply:
Removed, now